catalog/ftp/pom.xml

```diff
@@ -132,7 +132,7 @@
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.57</minimum>
+                                            <minimum>0.55</minimum>
```

Comment:
Thanks! I wish we could increase them to our standard minimum of 75% though. Any idea why we can't get there, i.e. do we know what's not getting covered, etc.?